modules/programs/qutebrowser.nix

```
@@ -260,16 +260,20 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
+  config = let
+    qutebrowserConfig = if pkgs.stdenv.hostPlatform.isDarwin then
+      ".qutebrowser/config.py"
+    else
+      ".config/qutebrowser/config.py";
```

```suggestion
      "${config.xdg.configHome}/qutebrowser/config.py";
```

The `.config` folder is not guaranteed to be the user's `$XDG_CONFIG_HOME` 😃